docker/e2e/chaos.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash 
+RED='\033[0;31m'
+NC='\033[0m'
+CYAN='\033[0;36m'
+PURPLE='\033[0;35m'
+GREEN='\033[0;32m'
+
+c=0
+no_sleep () {
+    echo -e "${GREEN}RUN TENDERMINT NO SLEEP CHAOS TEST ${GREEN}"
+    while [ $c -le 14 ]
+    do
+    echo "${RED}STOP TENDERMINT NODE 0 $c times${RED}"
+    ((c++))
+    docker stop e2e-tendermint-0-1 
+    echo "${GREEN}START TENDERMINT NODE 0 $c times${GREEN}"
+	  docker start e2e-tendermint-0-1
+
+    echo "${RED}STOP TENDERMINT NODE 1  $c times${RED}"
+    docker stop e2e-tendermint-1-1 
+    echo "${GREEN}START TENDERMINT NODE 1  $c times${GREEN}"
+    docker start e2e-tendermint-1-1 
+
+    echo "${RED}STOP TENDERMINT NODE 2  $c times${RED}"
+    docker stop e2e-tendermint-2-1 
+    echo "${GREEN}START TENDERMINT NODE 2  $c times${GREEN}"
+    docker start e2e-tendermint-2-1 
+
+    echo "${RED}STOP TENDERMINT NODE 3  $c times${RED}"
+    docker stop e2e-tendermint-3-1 
+    echo "${GREEN}START TENDERMINT NODE 3  $c times${GREEN}"
+    docker start e2e-tendermint-3-1 
+
+    echo "${NC}NO SLEEP TEST OVER${NC}"
+
+  done
+    rapid_fire
```

Comment:
This line means you are running the `rapid_fire` tests twice?

Reply:
It only runs once, when no_sleep ends and same with random_intervals test they all run once; consecutively

Reply:
It's also called at line 120, right?.